migration/20160719-fix-incorrectly-encoded-work-descriptions.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
```

Comment:
Work.summary_text is the only place we set work_description, so for simplicity's sake I wrote the migration script to focus on summary_text. This means we don't have to call Work.calculate_presentation() or anything.